app/client/components/transferAccount/StandardTransferAccountList.tsx

```diff
@@ -265,6 +284,12 @@ class StandardTransferAccountList extends React.Component<
           onSelectChange={(s: React.Key[], u: React.Key[], a: boolean) =>
             this.onSelectChange(s, u, a)
           }
+          paginationOptions={{
+            currentPage: this.state.page,
```

Comment:
on initial page load, the current pagination page option wasn't being selected. adding `currentPage` fixes this